docs/product-assembly_pam-on-openshift/src/main/asciidoc/environment-authoring-con.adoc

```diff
@@ -0,0 +1,13 @@
+[id='environment-authoring-con']
+= Authoring environment
+You can deploy an environment for creating and modifying processes using {CENTRAL}. It consists of {CENTRAL} for the authoring work and {KIE_SERVER} for test execution of the processes.
+
+Depending on your needs, you can deploy either a single authoring environment or a high-availability (HA) authoring environment. 
+
+A single authoring environment contains two pods. One of the pods runs {CENTRAL}, the other runs {KIE_SERVER}. The {KIE_SERVER} includes an embedded H2 database engine. This type of environment uses the least possible amount of resources.
+
+image::Authoring.png[Architecture diagram of a single authoring infrastructure]
+
+An HA authoring environment contains several pods. Both {CENTRAL} and {KIE_SERVER} are provided in scalable pods that can run in parallel and share persistent storage. The database is provided by a separate high-availability service. Use a high-availability authoring environment to provide maximum reliability and responsiveness, especially if several users are onvolved in authoring at the same time.
+
+image::AuthoringHA.png[Architecture diagram of a high-availability authoring infrastructure]
```

Comment:
In this image are arrows to elastic search different then others. Also some arrows are over the other arrows and it can be confusing (mid-BC -> elastic search and mid-BC -> openshift service), can you fix it?

Reply:
I can't, but the image was changed now, and in any event I will likely have to ask designers for a review of this one. @bmozaffa could you provide the source for your diagrams?